DateTime.py

```diff
@@ -26,7 +26,7 @@ def handleQuery(query):
     if fields:
         def makeItem(text: str, subtext: str):
             return Item(
-                id=__prettyname__,
+                id=text,
```

Comment:
This id not a good idea. The id is used for the ranking and should be somewhat persitent. Using the timestamp effectively messes up the MRU score cache.